website/static/css/pages/share-buttons.css

```diff
@@ -0,0 +1,37 @@
+.share-buttons {
+    margin: 3px 20px;
+}
+
+.share-buttons a {
+    padding: 0px 2px;
+    display: inline-block;
+}
+
+.share-buttons .fa {
+    color: #08c;
+    font-size: 1.6em;
+    line-height: 1.6em;
+}
+
+#share .share-buttons {
```

Comment:
The "share" element is defined in filepage/index.js:53. Those css rules are in fact necessary unlike some of the others!
m('li.active', m('a[href="#share"][data-toggle="tab"]', 'Share')),